pages/api/cardinfo/program.ts

```diff
@@ -0,0 +1,33 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { ResponseUtil } from "@utils/responseUtil";
+import {
+    getProgramCardInfos,
+    getProgramCardInfo,
+} from "@database/programcardinfo";
+
+/**
+ * handle takes the programId parameter and returns
+ * the program associated with the programId
+ * @param req API request object
+ * @param res API response object
+ */
+export default async function handle(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    if (req.method == "GET") {
```

Comment:
I think this functionality is really similar to `api/program/[id].ts`. I think you should just use that and send everything to the frontend. Just let the frontend process the data in the format that it needs to be in.

Reply:
Where do we put the existing getProgram then? Does that mean we don't need any endpoint to get a Program object, every GET would get a ProgramCardInfo?

Reply:
Like we discussed on Slack, I think we don't have to have an endpoint to get a Program object since we always want to get the program along with the language. Let me know if I'm mistaken